amqp/json_schemas/message.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://asyncapi.com/amqp/bindings/message.json",
+  "title": "Message Schema",
+  "type": "object",
+  "additionalProperties": false,
+  "patternProperties": {
+    "^x-[\\w\\d\\.\\-\\_]+$": {
+      "$ref": "https://github.com/asyncapi/asyncapi-node/blob/master/schemas/2.0.0.json#/definitions/specificationExtension"
+    }
+  },
+  "properties": {
+    "contentEncoding": {
+      "type": "string"
+    },
+    "messageType": {
+      "type": "string",
+      "default": "latest"
+    },
+    "bindingVersion": {
+      "type": "string",
+      "default": "latest"
```

Comment:
please fix as I suggested in another comment, 
it shoudl not be here and the same is for all the other `bindingVersion` props